packages/notifications/src/styled/content/StyledTitle.tsx

```diff
@@ -6,19 +6,24 @@
  */
 
 import styled from 'styled-components';
-import classNames from 'classnames';
-import CalloutStyles from '@zendeskgarden/css-callouts';
 import { retrieveComponentStyles } from '@zendeskgarden/react-theming';
 
 const COMPONENT_ID = 'notifications.title';
 
 /**
- * Used for Notification titles. Supports all `<div>` props
+ * Used for Notification titles. Supports all `<div>` props.
+ * For improved semantics, pass an "h1" or "strong" to the `as` prop.
+ * Additional styles can be applied to the `as` element to override
+ * user agent stylesheets.
```

Comment:
Move this documentation to `/elements/Title` so that it shows in the public API prop sheet.